tests/integration/distribution/genesis_test.go

```
@@ -0,0 +1,188 @@
+	s.Require().NoError(err)
+	s.Require().Equal(rewards, outstanding.Rewards)
+
+	genesisState, err := s.app.ModuleManager.ExportGenesis(ctx)
+	s.Require().NoError(err)
+	stateBytes, err := json.MarshalIndent(genesisState, "", " ")
+	s.Require().NoError(err)
+
+	db := coretesting.NewMemDB()
+	conf2 := simtestutil.DefaultStartUpConfig()
+	conf2.DB = db
+	app2, err := simtestutil.SetupWithConfiguration(
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)